doc.json

```diff
@@ -45,6 +45,66 @@
           }
         }
       }
+    },
+    "/api/v1/auth/google": {
+      "get": {
+        "tags": [
+          "authentication"
+        ],
+        "summary": "User registration or login",
+        "description": "A Social platform for the creative at heart",
+        "consumes": [
+          "google api"
+        ],
+        "produces": [
+          "authorization token"
+        ],
+        "responses": {
```

Comment:
Please check if you can include parameters required in this Swagger doc.

Reply:
Thanks, Jide but I don't have parameters in this feature.